### ISSUE TITLE
Missing .meta files, all references in the assets are broken

### ISSUE THREAD

**Issue by nguyentrong101094** [AlphaMistral/Mistral-Trail#5]
The .meta files need to be included with the project. Currently every single prefab included in the asset have missing prefab and missing script error

**Comment by ndumais** [comment]
Yeah, same issue here, there's a good amount of script used (5 on the camera prefab) so it's a bit hard to figure out how all of this works, would happy to fork it and push a fix if anyone has the info on how to set this up
